### ISSUE TITLE
How to change buildkit config after build buildx create ?

### ISSUE THREAD

**Issue by abcfy2** [docker/buildx#1015]
I know buildx create has an option `--config` to set buildkit config.

But how do I change this config after `docker buildx create --use`?

Thanks.

**Comment by jsternberg** [comment]
I don't believe this is currently possible and my recommendation would be to create the container yourself and use the remote driver for this functionality. If more is desired, like adding an update function to the driver, then we can also consider that. Please open a new issue if that's the case.

I'm going to close this one for now. Thanks.
